content/company/testimonials.mdx

```diff
@@ -0,0 +1,19 @@
+---
+seo:
+  title: Testimonials
+  description: >-
+    Delighting clients worldwide with top-tier Microsoft solutions.
+    Testimonials, scalable services, 30+ years of expertise.
+  showBreadcrumb: true
+subTitle: "#### We go the extra mile to turn our clients into fans\n\nHere's a list of some of\_[our clients](https://www.ssw.com.au/SSW/Company/Clients/)' testimonials. From small business solutions to large multinational companies, SSW has made happy clients all over the world and we are proud to share some of our experiences with you.\n\n## About US\n\nSSW's Consulting Services have delivered best of breed Microsoft solutions for more than 1,000 clients in 15 countries. With 40 consultants in 5 countries, SSW's developers are some of the best in the business. We have many Microsoft Certifications, 5 MVPs, and a Microsoft Regional Director.\n\nWe deliver scalable and extensible custom solutions with the future in mind. Our solutions improve businesses' bottom lines, with real time savings and long term value. We will provide you with the competitive advantage you need.\n\nSSW Consulting has over 30 years of experience developing awesome Microsoft solutions that today build on top of Angular, React, Azure, TFS, SharePoint, Office 365, .NET, WebAPI, Dynamics 365 and SQL Server.\n\n[KNOW MORE](https://www.ssw.com.au/ssw/Company/AboutUs.aspx)\n"
```

Comment:
Is the `KNOW MORE` the correct way to link to somewhere else? Also this is using the v1 website link.